src/projection/node/create-projection-node.ts

```diff
@@ -893,7 +907,10 @@ export function createProjectionNode<I>({
             if (!this.parent || hasTransform(this.parent.latestValues))
                 return undefined
 
-            if (this.parent.target && this.parent.layout) {
+            if (
+                (this.parent.target || this.parent.targetDelta) &&
```

Comment:
Should we just check `targetDelta`? When do we expecting `target` to be there but not `targetDelta`?

Reply:
I think maybe if we check for relativeTarget instead it'll catch both but earlier in the frame